### ISSUE TITLE
Wave form render is not as expected

### ISSUE THREAD

**Issue by Nico3652** [SimformSolutionsPvtLtd/audio_waveforms#116]
When I'm displaying the ```WaveFileWaveforms``` widget after recording the wave result doesn't reflect the real form. 
The form after recording is 100% always the same :   

https://user-images.githubusercontent.com/50270977/196492220-6caf9470-b706-4f29-be8a-77b3c0b26f68.mov

Here is the sample code for this video (almost all code came from your example project) :

```
import 'dart:io';

import 'package:audio_waveforms/audio_waveforms.dart';
import 'package:flutter/material.dart';
import 'package:path_provider/path_provider.dart';

void main() {
  runApp(MyApp());
}


class MyApp extends StatelessWidget {

  @override
  Widget build(BuildContext context) {
    return MaterialApp(
        debugShowCheckedModeBanner: false,
        home: AudioWaveRecord()
    );
  }
}

class AudioWaveRecord extends StatefulWidget {
  const AudioWaveRecord({Key? key}) : super(key: key);

  @override
  State<AudioWaveRecord> createState() => _AudioWaveRecordState();
}

class _AudioWaveRecordState extends State<AudioWaveRecord> {
  late final RecorderController recorderController;
  final PlayerController playerController = PlayerController();
  bool recording = false;

  @override
  void initState() {
    recorderController = RecorderController()
      ..androidEncoder = AndroidEncoder.aac
      ..androidOutputFormat = AndroidOutputFormat.mpeg4
      ..iosEncoder = IosEncoder.kAudioFormatMPEG4AAC
      ..sampleRate = 44100
      ..bitRate = 48000;
    super.initState();
  }

  @override
  void dispose() {
    playerController.dispose();
    recorderController.dispose();
    super.dispose();
  }

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(
        centerTitle: true,
        title: Text("Wave Test"),
      ),
      body: Padding(
        padding: const EdgeInsets.all(15.0),
        child: Column(
          mainAxisAlignment: MainAxisAlignment.spaceEvenly,
          children: [
            if (playerController.playerState != PlayerState.stopped) ...[
              Column(
                crossAxisAlignment: CrossAxisAlignment.start,
                children: [
                  Row(
                    children: [
                      Text(
                        "Result Waves",
                        style: TextStyle(fontSize: 20),
                      ),
                      Spacer(),
                      GestureDetector(
                        onTap: () {
                          playerController.stopPlayer();
                        },
                        child: Icon(
                            Icons.delete
                        ),
                      ),
                    ],
                  ),
                  SizedBox(height: 20),
                  AudioFileWaveforms(
                      size: Size(300, 100),
                      margin: const EdgeInsets.symmetric(horizontal: 15),
                      playerController: playerController,
                      playerWaveStyle: const PlayerWaveStyle(
                        liveWaveColor: Colors.grey,
                        fixedWaveColor: Colors.black,
                        scaleFactor: 0.5
                      )
                  ),
                ],
              ),
            ],
            Column(
              crossAxisAlignment: CrossAxisAlignment.start,
              children: [
                Text(
                  "Recorder Waves",
                  style: TextStyle(fontSize: 20),
                ),
                SizedBox(height: 20),
                AudioWaveforms(
                  enableGesture: false,
                  size: Size(300, 100),
                  recorderController: recorderController,
                  waveStyle: const WaveStyle(
                    waveColor: Colors.black,
                    extendWaveform: true,
                    showMiddleLine: false,
                  ),
                  margin: const EdgeInsets.symmetric(horizontal: 15),
                ),
              ],
            ),
            Spacer(),
            Center(
              child: GestureDetector(
                onTap: onTap,
                child: Container(
                  color: recording ? Colors.red : Colors.blueGrey,
                  width: 200,
                  height: 40,
                  alignment: Alignment.center,
                  child: Text(
                    recording ? "Stop" : "Record",
                    style: TextStyle(color: Colors.white,fontSize: 20),
                  ),
                ),
              ),
            )
          ],
        ),
      ),
    );
  }

  Future<void> onTap() async {
    if(!recording) {
      Directory dir = await getTemporaryDirectory();
      String path = dir.path + "/" + "record.mp4";
      await recorderController.record(path);
    }
    else {
      String? filePath = await recorderController.stop(false);
      if(filePath == null)return;
      await playerController.preparePlayer(filePath);
      await playerController.startPlayer(finishMode: FinishMode.loop);
    }
    setState(() {
      recording = !recording;
    });
  }
}
```

When the file read by the player doesn't come from recording, the wave result is different but still not reflecting the real audio data.
I read all examples and parameters for classes/widgets but still can't find the correct way.  

Please how to set the correct wave visualization according a fixed width ? 

```
audio_waveforms: ^0.1.5+1
```

**Comment by ujas-m-simformsolutions** [comment]
This happens because we are directly calculating waveform without decoding the file. I'm going to a PR in some time which will fix this issue.

**Comment by Nico3652** [comment]
Why this is not happening with your example app in ```WaveBubble``` widget ? I copied/pasted your code and run it and after record the waves are not like mines.

 I reproduced the same logic record -> play (code above) but I'm not able to get the same result

**Comment by ujas-m-simformsolutions** [comment]
@Nico3652 This can be affected by bytes in your recording file and the demo recording file aren't similar.

**Comment by alphatero** [comment]
@ujas-m-simformsolutions Hi, really need this, please fix it. Thanks.

**Comment by massi-d** [comment]
Hello, do you know when you will make available the correction of this bug?

**Comment by ujas-m-simformsolutions** [comment]
@Nico3652, @alphatero @massi-d We have released `1.0.0` which fixes this issue so do check it out. And I'm going to close this issue for now but if you encounter any other issue, feel free to open a new one.
